modules/nodes-cluster-enabling-features-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-cli_{context}"]
+= Enabling feature sets using the CLI
+
+You can use the OpenShift CLI (`oc`) to enable feature sets for all the nodes in a cluster by editing the `FeatureGate` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+To enable feature sets:
+
+. Edit the Feature Gate CR, named `cluster`, in the `openshift-config` project:
++
+[source,terminal]
+----
+$ oc edit featuregate cluster -n openshift-config
+----
++
+.Sample Feature Gate custom resource
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster <1>
+....
+
+spec:
+  featureSet: TechPreviewNoUpgrade <2>
+----
++
+--
+<1> The name of the `FeatureGate` CR must be `cluster`.
+<2> Add the feature sets that you want to enable in a comma-separated list:
+* `IPv6DualStackNoUpgrade` to enable the dual-stack networking mode.
+* `TechPreviewNoUpgrade` to enable specific Technology Preview features.
```

Comment:
Small nit for these two bullets:
* `IPv6DualStackNoUpgrade` enables the dual-stack networking mode.
* `TechPreviewNoUpgrade` enables specific Technology Preview features.